### DISCUSSION TITLE
Request for Slack access

### DISCUSSION THREAD

**Discussion by gezquinndesign** [accounts-js/accounts#1134] [Q&A] [answered]
Hello,

I don't seem to be having much luck interacting with anyone in the discussions here. I would really like some guidance on how to set up the oauth strategies for twitter and facebook.  This is just a message to see if i could get access to the Slack community to see if I have more luck there.  Is this possible @pradel ?

Once I'm able to get it working for my own site, I'm happy to submit some PRs that update the documentation and plug the missing gaps (eg the non-existent oauth-facebook package).

**Comment by pradel** [comment, marked as answer]
Hey you can join with this link https://join.slack.com/t/accounts-js/shared_invite/zt-3lp86s9c-IxMxiD3Xcb9oPgzCDdfyOw

**Reply by gezquinndesign** [reply]
Thanks @pradel, I managed to figure out a working solution in the end.  I had to write my own implementations using the OAuthProvider interface.  For twitter, I had to add an additional method in order to obtain a request token.  I'm not sure if I'm missing something but I couldn't figure out a way to do it using the oauth-twitter package in it's current state.

Anyway, thank you for the invite. I'll let you know once I'm ready to add some PRs.

**Reply by pradel** [reply]
The twitter package was created some time ago it's possible that something changed regarding the twitter API that needs to be changed.

**Comment by darkbasic** [comment]
Slack is not being used anymore, please join our Discord server: https://discord.gg/nYSyrWPPdu
